workshops/pong_game/README.md

```diff
@@ -0,0 +1,213 @@
+---
+name: 'Pong Game'
+description: 'Classic Pong Game with Python'
+author: '@YashKalbande'
+img: 'https://cloud-awrat6ufv.vercel.app/0pong_game.gif'
+---
+
+
+# Pong Game
+
+**Pong Game** is one of the first computer games that was created in the world. Pong is a simple yet compelling **tennis** game. It features **two paddles** and a **ball**. The goal is to defeat your opponent. In this workshop, you'll learn how to build your own  **two-player** `Pong Game`. In this workshop, we are going to use libraries like `freegames`, `random`, and `turtle`. 
+# Demo
+
+Here's a [**demo**](https://repl.it/@YashKalbande/ponggame) of what we'll be making. You can see both the final code and test it out on this Repl environment.
+
+![pong game demo](https://cloud-awrat6ufv.vercel.app/0pong_game.gif)
+
+# Table of contents
+
+- [Pong Game](#pong-game)
+- [Demo](#demo)
+- [Table of Contents](#table-of-contents)
+- [Setup](#setup)
+- [Building The Game](#building-the-game)
+    - [1. Generate Random Values](#1-generate-random-values)
+    - [2. Declare Global Variables](#2-declare-global-variables)
+    - [3. Moving the Paddles](#3-moving-the-paddles)
+    - [4. Drawing our Paddle](#4-drawing-our-paddle)
+    - [5.Draw game and move Pong Ball](#5-draw-game-and-move-pong-ball)
+    - [6. The Final Touch!](#6-the-final-touch!) 
+- [Future Hacking](#future-hacking)
+
+# Setup
+
+### Setting up our Coding Environment
+![gif of human coding](https://cloud-awrat6ufv.vercel.app/1coding.gif)
+
+For this workshop, we're going to be using an online code editor called [Repl.it](https://repl.it/site/about).
+
+1. Head over to [Repl.it/languages/Python3](https://repl.it/languages/python3)
+   * This will create a new [Python 3](https://www.w3schools.com/python/) Repl for you to use to code.
+2. Edit your new `main.py` file with the following code:
+
+```python
+from random import choice, random
+from turtle import *
+from freegames import vector
+```
+* The code above essentially says the following:
+  * Import `choice` and `random` function from the module `random`. This is a built-in Python module that gets random values for our ball to move in random motion.
+  * Add all the functions from the `turtle` library. Learn more about the Turtle library [here](https://docs.python.org/3/library/turtle.html). It is a way to implement graphics in Python projects
+  * Get the [`vector`](http://www.grantjenks.com/docs/freegames/api.html#freegames.vector) functions from the `freegames` library. The [freegames library](http://www.grantjenks.com/docs/freegames/index.html) is a basic module with built-in functions that do standard game tasks like moving around and setting a playing field.
+
+# Building The Game
+
+### 1. Generate Random Values
+```python
+def value():
+    return (3 + random() * 2) * choice([1, -1])
+```
+`value` function will return us randomly generate value between `(-5, -3)` or `(3, 5)` each time. This will give **unpredictable direction** to ball.
```

Comment:
```suggestion
`value` function will return us randomly generate value between `(-5, -3)` or `(3, 5)` each time. This will give **unpredictable directions** to the ball each time.
```